Runtime/Core/AxisState.cs

```diff
@@ -219,11 +219,13 @@ public bool Update(float deltaTime)
             
             if (m_InputAxisProvider != null)
                 m_InputAxisValue = m_InputAxisProvider.GetAxisValue(m_InputAxisIndex);
+#if ENABLE_LEGACY_INPUT_MANAGER
             else if (!string.IsNullOrEmpty(m_InputAxisName))
             {
                 try { m_InputAxisValue = CinemachineCore.GetInputAxis(m_InputAxisName); }
                 catch (ArgumentException e) { Debug.LogError(e.ToString()); }
             }
+#endif
```

Comment:
This might not be desired, because it hides the fact that there is no correct input provider assigned!

Reply:
@glabute

Reply:
No, wait.  This change should be reverted, and instead change the default implementation of CinemachineCore.GetInputAxis